### ISSUE TITLE
Excessive LDAP log messages

### ISSUE THREAD

**Issue by gig13** [owncloud/core#11624]
**Expected behavior**
Suppress LDAP log messages that are not fatal when the log level is set to 4 (Fatal only). Report fatal errors with useful audit information so we can determine which user(s) are having issues.

**Actual behavior**
The log displays an excessive amount of bind failed:19: Constraint violation messages. These messages appear to be coming mostly from Windows Sync client users who's password's have recently changed, or are incorrect, or during network hiccups and the Windows sync client attempts to reconnect multiple times a second which locks their LDAP account. (An anti-DoS setting in LDAP server)

**Steps to reproduce**
1. Set the log level to 4 (Fatal only)
2. Setup a sync client for an LDAP user
3. Change password's for the LDAP user several times - observe the logs

**Server Configuration**
OS: RHEL6.6 x86_64
DB: MySQL 5.1.73-3.el6_5 
PHP:  php x86_64 5.3.3-38.el6 (Red Hat back ports security patches)

Log messages:
```
Oct 16 09:59:55 ldrive ownCloud[14461]: {user_ldap} Bind failed: 19: Constraint violation
Oct 16 09:59:55 ldrive ownCloud[14461]: {user_ldap} Bind failed: 19: Constraint violation
Oct 16 09:59:55 ldrive ownCloud[14809]: {user_ldap} Bind failed: 19: Constraint violation
Oct 16 09:59:56 ldrive ownCloud[14809]: {user_ldap} Bind failed: 19: Constraint violation
Oct 16 09:59:56 ldrive ownCloud[9618]: {user_ldap} Bind failed: 19: Constraint violation
Oct 16 09:59:56 ldrive ownCloud[9618]: {user_ldap} Bind failed: 19: Constraint violation
Oct 16 09:59:56 ldrive ownCloud[18988]: {user_ldap} Bind failed: 19: Constraint violation 
```

**Comment by MorrisJobke** [comment]
@blizzz @dragotin Wasn't there also a issue that the client doesn't know if the LDAP server has gone, the password has changed or the password is correct and something other happened?

**Comment by MorrisJobke** [comment]
https://github.com/owncloud/client/issues/2615

**Comment by MorrisJobke** [comment]
@blizzz The solution here is to catch erro 19 as well and don't log it?

**Comment by blizzz** [comment]
Honestly, I do not see a point to prevent it from being logged. Because next time, there will be a reporter complaining that user x cannot login and no information about the cause is written to the log file.

I can offer to move this from ERROR to WARN level, if that helps?

**Comment by cdamken** [comment]
@blizzz That was my wish list in https://github.com/owncloud/core/issues/9383 :smile:

**Comment by MorrisJobke** [comment]
Fix is here: #13848
